test-app/tests/integration/components/checkbox-field-test.gts

```diff
@@ -0,0 +1,223 @@
+/* eslint-disable no-undef -- Until https://github.com/ember-cli/eslint-plugin-ember/issues/1747 is resolved... */
+/* eslint-disable simple-import-sort/imports,padding-line-between-statements,decorator-position/decorator-position -- Can't fix these manually, without --fix working in .gts */
+
+import { click, find, render, setupOnerror } from '@ember/test-helpers';
+import { module, test } from 'qunit';
+
+import CheckboxField from '@crowdstrike/ember-toucan-core/components/form/checkbox-field';
+import { setupRenderingTest } from 'test-app/tests/helpers';
+
+module('Integration | Component | CheckboxField', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function (assert) {
+    await render(<template>
+      <CheckboxField @label="Label" @rootTestSelector="test" data-checkbox />
+    </template>);
+
+    assert.dom('[data-label]').hasText('Label');
+
+    assert
+      .dom('[data-hint]')
+      .doesNotExist(
+        'Expected hint block not to be displayed as a hint was not provided'
+      );
+
+    assert.dom('[data-checkbox]').hasTagName('input');
+    assert.dom('[data-checkbox]').hasAttribute('id');
+    assert.dom('[data-checkbox]').hasNoAttribute('aria-invalid');
+
+    assert
+      .dom('[data-error]')
+      .doesNotExist(
+        'Expected error block not to be displayed as an error was not provided'
+      );
+    assert
+      .dom('[data-root-field="test"] > [data-control]')
+      .hasNoClass('shadow-error-outline');
+  });
+
+  test('it renders with a hint', async function (assert) {
+    await render(<template>
+      <CheckboxField @label="Label" @hint="Hint text" data-checkbox />
+    </template>);
+
+    let hint = find('[data-hint]');
+
+    assert.dom(hint).hasText('Hint text');
```

Comment:
Definitely not!  Just copy pasta I think from the Field component.  I was using `find()` whenever I was also grabbing things like attributes off the element, but in this case here it can definitely use qunit-dom instead!  Thank you for catching this!